src/richie/apps/courses/templates/courses/cms/fragment_program_glimpse.html

```diff
@@ -1,30 +1,36 @@
 {% load i18n cms_tags extra_tags thumbnail %}
 {% comment %}Obviously, the context template variable "program" is required and must be a Program page extension{% endcomment %}
 {% with program_page=program.extended_object %}
-<a href="{{ program.extended_object.get_absolute_url }}" class="program-glimpse program-glimpse--link{% if program.extended_object.publisher_is_draft %} program-glimpse--draft{% endif %}" >
-    <div class="program-glimpse__media">
-    {% get_placeholder_plugins "program_cover" program_page as plugins or %}
-        <div class="program-glimpse__empty">{% trans 'Cover' %}</div>
-    {% endget_placeholder_plugins %}
-    {% blockplugin plugins.0 %}
-    <img src="{% thumbnail instance.picture 300x170 replace_alpha='#FFFFFF' crop upscale subject_location=instance.picture.subject_location %}"
-        srcset="
-        {% thumbnail instance.picture 300x170 replace_alpha='#FFFFFF' crop upscale subject_location=instance.picture.subject_location %} 300w
-        {% if instance.picture.width >= 600 %},{% thumbnail instance.picture 600x340 replace_alpha='#FFFFFF' crop upscale subject_location=instance.picture.subject_location %} 600w{% endif %}
-        {% if instance.picture.width >= 900 %},{% thumbnail instance.picture 900x510 replace_alpha='#FFFFFF' crop upscale subject_location=instance.picture.subject_location %} 900w{% endif %}
-        "
-        sizes="300px"
-        alt=""
-    />
-    {% endblockplugin %}
+<div class="program-glimpse{% if program.extended_object.publisher_is_draft %} program-glimpse--draft{% endif %}" >
+    <div class="program-glimpse__media" aria-hidden="true">
+        <a href="program-glimpse__stretched-link" tabindex="-1" href="{{ program.extended_object.get_absolute_url }}">
```

Comment:
You have 2 `href` here. Shouldn't this be `class="program-glimpse__stretched-link"`?